demos/storybook/stories/list-item-tag/within-an-InfoListItem.stories.ts

```diff
@@ -0,0 +1,19 @@
+import * as Colors from '@pxblue/colors';
+// import { InfoListItem, ListItemTag } from '@pxblue/angular-components';
```

Comment:
Please lint the demos/storybook folder.

Reply:
I was able to run prettier, but when kept getting an error when I ran yarn lint and yarn lint:fix.  May I have another set of eyes check this out.

Reply:
Got it fixed with commit 6918cb5